examples/05-fdem/plot_stitched_1d_bookpurnong.py

```diff
@@ -0,0 +1,276 @@
+
+"""
+Inversion of RESOLVE data acquired at Bookpurnong, Austrailia
+=============================================================
+
+XXX
```

Comment:
This would be good to fill in (or remote at this point), but even a sentence or 2 that points to EM geosci for more background would be great.